src/UIState/SeePHSlope.cpp

```diff
@@ -0,0 +1,12 @@
+/**
+ * SeePHSlope.cpp
+ */
+
+#include "SeePHSlope.h"
+
+#include "Devices/LiquidCrystal_TC.h"
+#include "Devices/PHProbe.h"
+
+void SeePHSlope::loop() {
+  LiquidCrystal_TC::instance()->writeLine(PHProbe::instance()->getSlope().c_str(), 1);
```

Comment:
Does the slope continually change?

Reply:
I agree that this should transfer over to `start()`. Perhaps do a search for `loop()` to see if we missed any others.

Reply:
Currently no. I'll change it to start()